gammapy/scripts/image_pipe.py

```diff
@@ -175,13 +222,135 @@ def make_maps(self, radius, bkg_norm=True):
             Disk radius in pixels for the significance map.
         bkg_norm : bool
             If true, apply the scaling factor to the bkg map
+        spectral_index : int
+            spectral index of the source spectrum
+        exposure_weighted_spectrum : bool
+            True if you want that the total excess / exposure gives the integrated flux
+        ncounts_min : int
+            Minimum counts required for one observation
         """
-        self.make_total_counts()
-        self.make_total_bkg(bkg_norm)
+        self.make_total_counts(ncounts_min)
+        self.make_total_bkg(bkg_norm, ncounts_min)
         self.make_significance(radius)
+        self.make_total_excess()
+        self.make_total_exposure(spectral_index, exposure_weighted_spectrum, ncounts_min)
 
     def make_psf(self):
         log.info('Making PSF ...')
 
-    def make_exposure(self):
-        log.info('Making EXPOSURE ...')
+    def make_total_excess(self):
+        """Compute excess between counts and bkg map."""
+        self.maps["total_excess"] = SkyMap.empty_like(self.empty_image)
+        excess = self.maps["total_counts"].data - self.maps["total_bkg"].data
+        self.maps["total_excess"].data = excess
+
+    def make_exposure(self, obs_id, spectral_index=2.3):
+        """Compute the exposure map for one observation. Excess/exposure will give the differential flux
```

Comment:
Put the sentence how flux and exposure are related on a separate line after a blank line.

Also, if you can put a formula how exposure is defined, I think in this case it would be good.
(it's the integral over flux x area x time assuming a PL)
See [here](https://gammapy.readthedocs.org/en/latest/api/gammapy.stats.significance.html#gammapy.stats.significance) as an example how to insert a formula in a docstring.